### PR TITLE
HexagonGrid: Factorize the bubble snap

### PR DESCRIPTION
**Pull request by D8H**
I thought I couldn't factorized it but I actually did a typo.
Tested with the BimBam example

### CODE REVIEW

File: Extensions/HexagonalGrid.json [hunk 1]
```diff
@@ -197,21 +197,12 @@
...
-          "parameterObjects": "Object",
-          "useStrict": true,
-          "eventsSheetExpanded": true
-        },
-        {
-          "disabled": true,
-          "folded": false,
```
Comment (Bouh) [inline Extensions/HexagonalGrid.json:214]: @D8H You are sure you don't use this part anymore?
I'am asking before merging.

Comment (D8H) [reply Extensions/HexagonalGrid.json:214]: I'm sure. The same JS code is in another action that is called in place.

**Comment by Bouh** [comment]
Thank for your feedback, the extension is now updated!

**Comment by 4ian** [comment]
Thanks both 👍

**Pull request merged**